service/src/test/java/bio/terra/workspace/pact/TpsApiTest.java

```diff
@@ -253,23 +248,21 @@ public RequestResponsePact linkPaoWhenBothExist(PactDslWithProvider builder) {
             .object("resultingPao");
 
     return builder
-        .given(existingPolicyState)
-        .given(secondPolicyState)
+        .given("a policy with the given id exists", Map.of("id", existingPolicyId.toString()))
+        .given("a policy with the given id exists", Map.of("id", secondPolicyId.toString()))
         .uponReceiving("A request to link the policies")
         .method("POST")
         .headers(contentTypeJsonHeader)
         .body(linkRequestShape)
-        .pathFromProviderState(
-            "/api/policy/v1alpha1/pao/$s/link".formatted(existingPolicyProviderStateValue),
```

Comment:
I just want to point out there was a `$s` here instead of a `%s`, which I assumed to be a bug, so I fixed it.

Reply:
yup, definitely a bug